### PR TITLE
app-editors/appflowy-bin: add libmpv dependency through mpv:0

### PR DESCRIPTION
**Pull request by mbaraa**
Reference #4810

### CODE REVIEW

**Comment by liangyongxiang** [comment]
@Linerre

**Comment by Linerre** [comment]
> @Linerre

我看着可以，只是不知道原来的打包者什么看法.
It looks good to me. Not sure what the original maintainer thinks.

File: app-editors/appflowy-bin/appflowy-bin-0.5.9.ebuild [hunk 1]
```diff
@@ -43,6 +44,8 @@ QA_PRESTRIPPED="
 QA_PREBUILT="*"
 
 src_install() {
+	dosym libmpv.so.1 libmpv.so.2
```
Comment (liuyujielol) [inline app-editors/appflowy-bin/appflowy-bin-0.5.9.ebuild]:
```suggestion
	dosym -r /usr/lib64/libmpv.so.1 /usr/lib64/libmpv.so.2
```

Comment (mbaraa) [reply app-editors/appflowy-bin/appflowy-bin-0.5.9.ebuild]: oopsie

**Comment by liuyujielol** [comment]
I don't actually use appflowy any more, feel free to make changes to this ebuild.

**Pull request merged**